modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/DefragmentationCommand.java

```diff
@@ -53,7 +54,7 @@ public class DefragmentationCommand extends AbstractCommand<DefragmentationArgum
     /** {@inheritDoc} */
     @Override public Object execute(GridClientConfiguration clientCfg, Logger log) throws Exception {
         try (GridClient client = Command.startClient(clientCfg)) {
-            Optional<GridClientNode> firstNodeOpt = client.compute().nodes().stream().filter(GridClientNode::connectable).findFirst();
+            Optional<GridClientNode> firstNodeOpt = anyConnectableServerNode(client.compute());
```

Comment:
The change is not equivalent: code 'before' was taking the first node, while 'after' it takes any node. Is this because `compute().nodes()` ordering does not have any specific sense and might be considered as arbitrary?

Reply:
From what I see in GridClientTopology implementation, current state of topology on a client side is managed by a HashMap<UUID, GridClientNodeImpl> collection that doesn't maintain any particular order of elements.

So I believe that in this case there is no semantic difference between "first node" and "any node" methods.